portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -683,6 +740,7 @@ definition {
 		property portal.acceptance = "false";
 
 		// TODO LPS-130511 CanBeCreatedFromAutosavedDraft pending implementation
+
```

Comment:
Hi @ewertonluna , I believe this line is an error on this test.

Reply:
Hi, @diogocordeiro! This new line was added after running `ant format-source-current-branch`. The SF command is adding this extra line. I'll look more into it tomorrow. Thank you